fastlane/lib/fastlane/actions/swiftlint.rb

```diff
@@ -11,8 +11,21 @@ def self.run(params)
           UI.user_error!("Your version of swiftlint (#{version}) does not support autocorrect mode.\nUpdate swiftlint using `brew update && brew upgrade swiftlint`")
         end
 
+        if params[:mode] == :autocorrect && version >= Gem::Version.new('0.43.0')
+          UI.deprecated("Your version of swiftlint (#{version}) has deprecated autocorrect mode, please start using fix mode in input param")
+          UI.important("For now, switching swiftlint mode `from :autocorrect to :fix` for you 😇")
+          params[:mode] = :fix
+        end
+
+        if params[:mode] == :fix && version < Gem::Version.new('0.43.0')
```

Comment:
These two will never happen together so I think it makes sense here to make this an `elsif`
```suggestion
        elsif params[:mode] == :fix && version < Gem::Version.new('0.43.0')
```